be/src/olap/snapshot_manager.h

```diff
@@ -99,6 +102,9 @@ class SnapshotManager {
     // snapshot
     Mutex _snapshot_mutex;
     uint64_t _snapshot_base_id;
+
+    // TODO(zxy) used after
+    std::shared_ptr<MemTracker> _mem_tracker = nullptr;
```

Comment:
In the next pr, this tracker will be switched to the tls mem tracker in the public func of `SnapshotManager`.

In this pr, I created all the trackers that will be used in the future, and built a complete mem tracker tree. (Perhaps it would be better to do this with a pr alone... = =||| )